v63003/outref/gtm8735.txt

```diff
@@ -24,8 +24,39 @@ Database file ##TEST_PATH##/mumps.dat now has BG access method
 %YDB-W-WCWRNNOTCHG, Not all specified database files were changed
   Access method                          BG  Global Buffers                1024
   LOCK shares DB critical section     FALSE  Read Only                      OFF
-# Displaying status of gtmhelp database
+# Displaying status of gtmhelp databases
+-r--r--r-- ##SOURCE_PATH##/dsehelp.dat
+-r--r--r-- ##SOURCE_PATH##/gdehelp.dat
+-r--r--r-- ##SOURCE_PATH##/gtmhelp.dat
+-r--r--r-- ##SOURCE_PATH##/lkehelp.dat
+-r--r--r-- ##SOURCE_PATH##/mupiphelp.dat
+#Displaying status of ##SOURCE_PATH##/dsehelp.gld
+  LOCK shares DB critical section     FALSE  Read Only                       ON
+%YDB-E-DBFILOPERR, Error doing database I/O to region ##SOURCE_PATH##/dsehelp.dat
```

Comment:
Need a comment line here describing what is being attempted (MUPIP SET). Same before the output where a SET is attempted.